nix/pkgs/extensions/cargo-project.nix

```diff
@@ -0,0 +1,165 @@
+{ stdenv
+, lib
+, makeRustPlatform
+, pkg-config
+, protobuf
+, sources
+, channel
+, pkgs
+, clang
+, libxfs
```

Comment:
is this needed?
please trim down any unused inputs

Reply:
Thanks, Removed